app/lib/tool/test_profile/importer.dart

```diff
@@ -112,6 +112,13 @@ Future<void> importProfile({
               replacedBy: testPackage.replacedBy,
               isUnlisted: testPackage.isUnlisted,
             ));
+
+        if (testPackage.retractedVersions != null) {
+          testPackage.retractedVersions!.forEach((version) async {
```

Comment:
Let's do a for loop over the versions, otherwise the async blocks may not complete until they are used e.g. in tests:

```
for (final version in testPackage.retractedVersions!) {
  await client.setVersionOptions(packageName, version, VersionOptions(isRetracted: true));
}
```

Reply:
Good point, done.